aws_analysis_tools/ec2_events/flowdock_listener.py

```diff
@@ -33,15 +34,24 @@ def add_flowdock_listener_cli_arguments(parser):
     group.add_argument(
         '--flowdock-token',
         type=str,
-        help="Flowdock API token. If provided, will post to Flowdock",
+        help="Flowdock API token. If provided, will post to Flowdock. (default: %(default)s)",
         default=None,
     )
 
+    group.add_argument(
+        '--urgent',
+        type=int,
+        help="If the maintenance event is closer than this hours, it is notified to members. (default: %(default)s)",
```

Comment:
nit/suggested clarification: "If the event is closer than this number of hours, send an `@team` notification."